data/sql/updates/pending_db_world/rev_1555017745656363300.sql

```diff
@@ -0,0 +1,83 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1555017745656363300');
+
+-- Item should only drop from Riverpaw Mongrel (123)
+DELETE FROM `reference_loot_template` WHERE `Item`=821;
```

Comment:
I mean for these, miss click in previous comment :smile:

Reply:
You're right, tomorrow i will be home for that

Reply:
I think the statements are fine as they are, easier to read and better documented. No need to merge it into one statement in my opinion.